src/components/BookInfo.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@200;400;600&display=swap');
+
+.Info-Box {
+  flex: 3;
+}
+
+.Cat-Info {
+  display: block;
+  font-family: Montserrat, serif;
+  padding: 2.5px 10px;
+  font-size: 0.975rem;
+  color: #121212;
+  opacity: 0.5;
+  font-weight: bold;
+}
+
+.Tit-Info {
+  display: block;
+  font-family: 'Roboto Slab', serif;
```

Comment:
**Changes Required**
I must say your CSS files are well organized and modular, bravo. However;
- [ ] The CSS selector property `font-family` with the value of `Roboto Slab, serif` is repeated multiple times. You want to follow best CSS practice of Don't Repeat Yourself. (DRY). Please let us refactor it by creating for instance a `.family-roboto` in your `.jsx` file and having a `. family-robot{ font-family: 'Roboto Slab', serif;}` in your CSS file. Please do this for all repeating Selector properties in all your css files.